src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -298,8 +328,8 @@ public Path generate(Logger log) throws KaitaiException {
 
 		log.info("Kaitai: check version");
 		execute(this
-			.process(log)
-			.withArg("--version")
+				.process(log)
+				.withArg("--version")
```

Comment:
Please revert the indentation changes
```suggestion
			.process(log)
			.withArg("--version")
```